src/openrct2/game.c

```diff
@@ -1124,6 +1124,9 @@ void game_fix_save_vars()
 
     // Fix gParkEntrance locations for which the map_element no longer exists
     fix_park_entrance_locations();
+
+    // Fix disconnected entrances
```

Comment:
This comment isn't necessary. Rename the function to `fix_disconnected_entrances` if it isn't clear enough.